app/src/main/java/com/khinthirisoe/bakingapp/ui/baking/view/BakingViewHolder.kt

```diff
@@ -0,0 +1,28 @@
+package com.khinthirisoe.bakingapp.ui.baking.view
+
+import android.view.View
+import com.khinthirisoe.bakingapp.R
+import com.khinthirisoe.bakingapp.data.model.Recipe
+import com.khinthirisoe.bakingapp.ui.base.BaseViewHolder
+import kotlinx.android.synthetic.main.list_baking.view.*
+
+class BakingViewHolder(itemView: View) : BaseViewHolder(itemView) {
+
+    fun configureUI(baking: Recipe, position: Int) {
```

Comment:
@KhinThiriSoe where did onBind method go? onBind method sounds better. Well, call it `bind`. That's better than `configureUI`